src/Symfony/Component/Config/Loader/LoaderInterface.php

```diff
@@ -26,7 +26,7 @@ interface LoaderInterface
      *
      * @throws \Exception If something went wrong
      */
-    public function load($resource, $type = null);
+    public function load($resource, ?string $type = null);
```

Comment:
`?` should be removed (same below when null is the default), per our CS

Reply:
done